### PR TITLE
Bankroll

### PR DESCRIPTION
**Pull request by AMess33**
lots left, but figured your notes might help me move forward

### CODE REVIEW

**Comment by AMess33** [comment]
created a bankroll header to be displayed on all pages at the top. That will need some add and subtract buttons in order to update without playing sessions. 

created the bankroll page, which will display the bankroll header then a graph below it. need to figure out how to implement the graph and i think i want two different graphs, one that shows bankroll change over time, and another that will show +/- of all sessions with 0 being the starting point based on timeframe you want to view (month/ 6 months/ 12 months/ ytd). I do not want two graphs, just one graph but with the ability to show all these different views. 

creating the previous session containers that will just give you a list of all your previous session that have been completed below the graph view. quick info with Date of session, name of game of session played, and your +/- for that session. maybe a click to expand feature on those for notes/casino/location info. 

a lot of the displayed content on this page will just be database entries so i guess i'm just not sure how detailed to get at this point in the build since all the placeholder text i'm working with will be replaced with prop values.

**Comment by aautem** [comment]
@AMess33 Nice, sounds like a good plan. I'd go ahead and merge this in after resolving any issues and then start on the graphs in a new branch. Added a new ticket for that: https://github.com/AMess33/PokerRoll/issues/14

Can do the same for the previous session list. You're right, the placeholder data is just going to get replaced eventually but it's worth getting the graphs and previous session lists up and running with some fake data to start with. That way you can get your data types locked in so that when you go to build out your API you know exactly what data the front-end needs to power those graphs and lists.

File: apps/poker-roll/src/app/bankrollHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material';
```
Comment (aautem) [inline apps/poker-roll/src/app/bankrollHeader.jsx]: Double check your import here

File: apps/poker-roll/src/app/bankrollPage.jsx [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
+    return (
+        <div>
+            <Container maxWidth="sm">
+                <Box>
+                    <h4>Date</h4>
+                    <p>GAME Played</p>
+                    <span>+- result</span>
+                </Box>
+            {/* show info of previous sessions */}
+            {/* date/ location/ game type/ plus/minus of session */}
```
Comment (aautem) [inline apps/poker-roll/src/app/bankrollPage.jsx:16]: I'd just start basic with the previous session list, can come back later and add in the expand functionality to show more details.

File: apps/poker-roll/src/app/startSession.jsx [hunk 3]
```diff
@@ -173,23 +173,21 @@ const NewSession = (props) => {
...
+                value={gameType}
+                exclusive
+                onChange={handleToggleChange}
+                aria-label="Platform"
+              >
+                <ToggleButton value="cashgame">Cash Game</ToggleButton>
+                <ToggleButton value="tournament">Tournament</ToggleButton>
+              </ToggleButtonGroup>
+                {/* change which stack is show depending on cashgame or tournament is selected */}
+                { gameType === "cashgame" ? <CashGameForm/> : <TournamentForm/>}
```
Comment (aautem) [inline apps/poker-roll/src/app/startSession.jsx:188]: Noticed I couldn't seem to access these forms anymore, not sure if that was intentional for the time being while you're moving things around.

File: apps/poker-roll/src/app/homePage.jsx [hunk 4]
```diff
@@ -25,8 +26,8 @@ const Home = (props) => {
                 <div>
                     <h1>Poker Roll</h1>
                 </div>
-                <div>
-                    <SessionForm />
+                <div className='mainContent'>
```
Comment (aautem) [inline apps/poker-roll/src/app/homePage.jsx:29]: Is this className doing anything? Or planning on doing something with it going forward?

File: apps/poker-roll/src/app/startSession.jsx (hunk 3 above)
Comment (AMess33) [reply apps/poker-roll/src/app/startSession.jsx:188]: those are intended to be a modal/pop up when you click on start new session from the home screen, that functionality has not been built yet, so they are currently inaccessible

File: apps/poker-roll/src/app/bankrollHeader.jsx (hunk 1 above)
Comment (AMess33) [reply apps/poker-roll/src/app/bankrollHeader.jsx]: i had a bunch of other things included with button and wasn't using them so i got rid of them, good eye

File: apps/poker-roll/src/app/homePage.jsx (hunk 4 above)
Comment (AMess33) [reply apps/poker-roll/src/app/homePage.jsx:29]: yea thats just a placeholder for future styling to be applied to anything between the header and the nav bar at the bottom of the app.

**Pull request merged**